### PR TITLE
Upgrade to Micronaut 3.0

### LINKED ISSUE #171: Release a new version compatible with Micronaut 3.0
Please follow the instructions in https://github.com/micronaut-projects/micronaut-core/issues/5607 and update that core issue with the information related to the upgrade.

### PR DESCRIPTION
**Pull request by ilopmar**
Fixes #171

### CODE REVIEW

**Comment by ilopmar** [comment]
Of course all the test pass locally but fail on CI :sob:

File: liquibase/build.gradle [hunk 1]
```diff
@@ -10,6 +10,7 @@ dependencies {
     }
     api "io.micronaut:micronaut-management"
     api "io.micronaut.sql:micronaut-jdbc"
+    api "io.projectreactor:reactor-core"
```
Comment (sdelamo) [inline liquibase/build.gradle:13]: this should be `implementation` instead of `api`.

**Review by sdelamo — CHANGES_REQUESTED** [review]

**Review by sdelamo — APPROVED** [review]

**Comment by graemerocher** [comment]
Is there something specific this module needs that requires compiling against 3.0?

**Comment by ilopmar** [comment]
No, it's not. But I think sooner or later we need to upgrade all modules to 3, so if this works without problems and we've already upgraded it, does it hurts to keep it?
I can rollback the change and compile against 2.5.9 if we want, it's not a problem at all :-)

**Comment by graemerocher** [comment]
As discussed in the meeting every module that requires another downstream release adds to the work @jameskleeh is going to have to do to release everything 

It seems unnecessary to me to add to that work but it is up to @jameskleeh he is the one that will do the release

**Comment by ilopmar** [comment]
Yes, you are right. I can rollback to 2.5.9.

**Pull request merged**